_episodes/05-function-style.md

```diff
@@ -201,37 +234,51 @@ $ pip install yapf
 Now we can use yapf on our python files. Just to see the power of yapf, let's mangle one of our functions and use yapf to reformat it.
```

Comment:
Add a section on rewriting the function here. It should include renaming the variable to be more descriptive and an exercise to re-do the `calculate_distance` function.